src/share/backup/ob_archive_path.cpp

```diff
@@ -362,6 +362,19 @@ int ObArchivePathUtil::get_piece_info_file_path(const ObBackupDest &dest, const
   return ret;
 }
 
+// oss://[user_specified_path]/file_info.obarc
```

Comment:
rename input parameter dest to  archive_piece_dest

Reply:
should keep consistent with existing implemetation?